### DISCUSSION TITLE
Bi-directional sync (Airtable) [Apr 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5605] [Discord Threads]
> 🙋‍♂️ **Donut**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Not supported. We will be happy to understand your use case though  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1090841988103942214)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
